gemp-lotr/gemp-lotr-async/src/main/web/src/updated/jCard/modules/image-resolution/data/players-committee/index.ts

```diff
@@ -0,0 +1,7 @@
+import { updatesFrom2021 } from "./2021";
+import { updatesFrom2022 } from "./2022";
+
+export const playerCommitteeCards = [
```

Comment:
and here we re-bundle the various updates from each year and provide a single hook to grab them all!